lib/rubocop/cop/rspec/subject_stub.rb

```diff
@@ -56,25 +58,19 @@ class SubjectStub < Cop
         #     expect(foo).to receive(:bar).with(1)
         #     expect(foo).to receive(:bar).with(1).and_return(2)
         #
-        #   @example source that not matches
-        #     expect(foo).to all(receive(:bar))
-        #
         def_node_matcher :message_expectation?, <<-PATTERN
-          {
-            (send nil? :allow (send nil? %))
-            (send (send nil? :expect (send nil? %)) :to #expectation?)
-          }
+          (send
+            (send nil? { :expect :allow } (send nil? %))
```

Comment:
Wonder whether we should extend this to all expectations, using the Language module. Then we may need to add the stubs there as well

Reply:
For this specific case when we check if `subject` is stubbed, `expect/allow_any_instance_of` won't come into play. Is there anything else?

Reply:
Didn't see it in the wild, but `is_expected.to receive(:message)`

Reply:
Fair enough, pretty possible use case.